src/cli/process.c

```diff
@@ -32,7 +32,7 @@ enum {
     ACT_SKIP
 };
 
-static int process_one_crash(problem_data_t *problem_data)
+static int process_one_crash(problem_data_t *problem_data, int flags)
```

Comment:
If you want to reuse CMD_REPORT_* members (definitely good idea), you should name the parameter "cmd_flags" because I can imagine similar flags for the status and info commands.